src/js/front-end/actions/zones/general.js

```diff
@@ -10,8 +10,12 @@ import { addAbilityCounter } from "../counters/ability-counter.js";
 import { hideCard, revealCard } from "../general/reveal-and-hide.js";
 import { moveCard } from "../move-card-bundle/move-card.js";
 import { shuffleZone } from "./shuffle-zone.js";
+import { replayBlock } from "../../setup/general/replay-block.js";
 
-export const shuffleAll = (user, initiator, zoneId, indices, emit = true) => {
+export const shuffleAll = (user, initiator, zoneId, indices, emit = true, isFromReplay = false) => {
+    if (replayBlock('action','shuffleAll',isFromReplay)){
```

Comment:
i think the buttons and keybinds that enable shuffleAll, shuffleBottom etc. should be hidden/disabled during replay mode. then, you don't need to do this check for all of the functions (since the user should have no way to trigger it in replay mode)

Reply:
Done in [91bbd5e](https://github.com/xxmichaellong/ptcg-sim/pull/19/commits/91bbd5ef3e30bfa44b35e3eb347ad2ad2d770186), I left the VSTAR and GX buttons since they also serve as indicators